website/src/app/(home)/widgets/page.css

```
@@ -0,0 +1,183 @@
+  background-color: #f3f4f6;
+}
+.features-container .content-container {
+  margin-top: -8rem;
+}
+.dark .features-container {
+  background-color: #000;
+}
+.headline {
+  display: inline-flex;
+  font-size: 3.125rem;
+  font-size: min(4.375rem, max(8vw, 2.5rem));
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)